Reformatting

alazymeme opened the pull request:
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable

RAnders00 commented:
Running `reformat.sh` myself, I cannot replicate that it wants to reformat it like this

Are you sure your venv is up-to-date? (`rm -r venv && ./scripts/venvinstall.sh --dev`)

alazymeme commented:
> Running `reformat.sh` myself, I cannot replicate that it wants to reformat it like this
> 
> Are you sure your venv is up-to-date? (`rm -r venv && ./scripts/venvinstall.sh --dev`)

Just removed and updated the dependencies, doesn't look like it wants to change the format anymore.

Weird

RAnders00 commented:
There was a dependabot update for `black` recently
